yb-voyager/cmd/exportDataStatus.go

```diff
@@ -262,7 +262,7 @@ func startExportPB(progressContainer *mpb.Progress, mapKey string, quitChan chan
 				time.Sleep(100 * time.Millisecond)
 				break
 			} else if err != nil { //error other than EOF
-				utils.ErrExit("Error while reading file %s: %v", tableDataFile, err)
+				utils.ErrExit("Error while reading file %s: %v", tableDataFile.Name(), err)
```

Comment:
same for this as above
 utils.ErrExit("Error while reading file: %s: %v", tableDataFile.Name(), err)